packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -243,6 +244,7 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
     private readonly subLogger: ITelemetryLogger;
     private readonly thresholdOpsCounter: ThresholdCounter;
     private static readonly pendingOpsCountThreshold = 1000;
+    public visibilityState: VisibilityState = VisibilityState.NotVisible;
```

Comment:
I'd remove default value here, and make sure constructor assigns one always (and compiler checks that). That way all the code / values are in one place to review, easier (in the future) to review logic / see what values are assigned and when